sdcm/nemesis.py

```diff
@@ -240,6 +242,9 @@ def __init__(self, tester_obj, termination_event, *args, nemesis_selector=None,
         self.cluster: Union[BaseCluster, BaseScyllaCluster] = tester_obj.db_cluster
         self.loaders = tester_obj.loaders
         self.monitoring_set = tester_obj.monitors
+        nemesis_thread_name = f"Nemesis{uuid4()}"
+        self.actions_log = get_action_logger(source=nemesis_thread_name)
+        self._with_action_scope = self.actions_log.action_scope
```

Comment:
I see not reason to make this protected, I think `self.with_action_scope` will read better

Reply:
renamed with `self.action_log_scope`